### ISSUE TITLE
PhantomJS Issues

### ISSUE THREAD

**Issue by maureeno20** [addyosmani/a11y#58]
I get the following error when trying to run the tests:

Command failed: /usr/local/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs /usr/local/lib/node_modules/a11y/audits.js {"delay":1,"url":"http://bbc.co.uk","width":1024,"height":768} --ignore-ssl-errors=true --ssl-protocol=tlsv1 --local-to-remote-url-access=true
Audit failed

I got a similar error on initial install so I manually installed PhantomJS but still can't get any test to work.

**Comment by rickydazla** [comment]
Same here

**Comment by sjurlur** [comment]
Same here

**Comment by addyosmani** [comment]
Can you confirm what version of PhantomJS you have locally installed?

**Comment by rickydazla** [comment]
1.9.1.7

**Comment by sjurlur** [comment]
npm ls phantomjs
a11y@0.4.1 /usr/local/lib/node_modules/a11y
└── phantomjs@2.1.7

**Comment by jatinsi** [comment]
Try uninstalling phantomjs and a11y and reinstalling only a11y. 

>npm uninstall a11y --global
>npm uninstall phantomjs --global

>npm install --global a11y

I fixed the same issue on my machine using the above step.

**Comment by agarzola** [comment]
I’m getting this same error when trying to run `a11y` on certain websites, but not all. I’ve yet to find a definitive factor that predictably determines whether `a11y`ing a site will result in this error or not. I do know that the sites where it does fail, it does so consistently.

`http://bbc.co.uk`, as reported by @maureeno20, gets the error. So does `http://asmallbatch.co`. Both of those locations have redirects, but so does `http://mrkp.co` and that one loads fine.

Reinstalling `a11y` globally does not seem to have an effect, in my case. The only version of `phantomjs` I have installed (globally, anyway) is the one installed as a dependency of `a11y`:

```
$ npm ls --global phantomjs
/Users/agarzola/.nvm/versions/node/v4.2.4/lib
└─┬ a11y@0.4.1
  └── phantomjs@2.1.7
```

**Comment by jonnyi** [comment]
@agarzola Seeing the exact same issues even following the reinstallation step for a11y and phantomjs.

**Comment by bennypowers** [comment]
It looks like this is related to https://github.com/ariya/phantomjs/issues/12697 and https://github.com/addyosmani/a11y/blob/master/audits.js#L104-L109
